src/fourier.jl

```diff
@@ -1,53 +1,57 @@
 export
-    SpectralConv1d,
+    SpectralConv,
     FourierOperator
 
-struct SpectralConv1d{T, S}
+struct SpectralConv{T, S, N}
```

Comment:
```suggestion
struct SpectralConv{N, T, S}
```

It is better to put dims `N` in the front. This way it can dispatch to method args of type `SpectralConv{N}`.

Reply:
Done